src/clj_statsd.clj

```diff
@@ -46,43 +47,50 @@
 (defn publish
```

Comment:
Can you make publish a 3 and 4 arity function, with a default value of `[]` for tags?